nsist/spyder.nsi

```diff
@@ -38,6 +39,7 @@ SetCompressor lzma
 
 ; UI pages
 [% block ui_pages %]
+;!define MUI_PAGE_CUSTOMFUNCTION_RUN validate_running
 !define MUI_PAGE_CUSTOMFUNCTION_PRE validate_pre_install
```

Comment:
Maybe we could define a single validation function that calls the new `validate_running` and `validate_pre_install`, something like `pre_validations` @jsbautista ?